pages/peterportal/getting-started.mdx

```diff
@@ -0,0 +1,84 @@
+import { Steps } from "nextra/components";
+
+# Getting Started
+
+This document will guide you through setting up the project on your local machine, from cloning the repository to running the project locally. Whether you're a committee member or an open-source contributor, you'll find all the necessary steps to get started below.
+
+## Setup
+
+<Steps>
+
+### Cloning the Repository
+
+#### For Committee Members
+
+1. Clone the repository using the following command:
+
+   ```bash
+   git clone https://github.com/icssc/peterportal-client
+   ```
+
+2. Navigate into the cloned repository:
+   ```bash
+   cd peterportal-client
+   ```
+
+#### For Open Source Contributors
+
+1. Fork the project on GitHub by clicking the fork button in the top right, above the "About" section.
+2. Clone your forked repository to your local machine using the following command (replace `your username` with your GitHub username):
+   ```bash
+   git clone https://github.com/yourusername/peterportal-client
+   ```
+3. Navigate into the cloned repository:
+   ```bash
+   cd peterportal-client
+   ```
+
+### Installing Dependencies
+
+1. Ensure you have Node.js version 18 or 20 LTS installed. If not, consider using nvm for version management.
+2. Install all node dependencies by running:
+   ```bash
+   npm install
+   ```
+
+### Setting up Environment Variables
+
+1. Inside the `api` directory, rename the `.env.example` file to `.env`
+2. Add additional variables as needed for features requiring authentication or database access.
+
+### Switching Branches
+
+Before starting your work, switch to a new branch with a relevant name to your task:
+
+```bash
+git checkout -b [branch name]
```

Comment:
Syntax highlighting looks a little weird here. I think replacing the space with a hyphen will fix that.

```suggestion
git checkout -b [branch-name]
```

![image](https://github.com/icssc/docs/assets/8922227/a0052b86-0dee-4226-b6cc-ed8cc22d1afe)